src/opower/utilities/base.py

```diff
@@ -39,6 +39,11 @@ def accepts_mfa() -> bool:
         """Check if Utility implementations supports MFA."""
         return False
 
+    @staticmethod
+    def is_dss() -> bool:
+        """Check if Utility using DSS version of the portal."""

```

Comment:
Do you know what DSS stands for?

Reply:
Not sure for 100% but I think it's Digital Self Service. Some basic information is here: https://docs.oracle.com/en/industries/energy-water/digital-self-service/transactions-seamless/Content/Transactions/SeamlessIntegration/Introduction_Seamless.htm

Reply:
Thanks. Separately we should rename subdomain to utilitycode in UtilityBase.